hack/docker-install.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+# shellcheck disable=2046,2086
+docker_install()
```

Comment:
We need to detect podman systems and avoid docker installs. Very simply if podman CLI exists then we can skip installing docker.

Reply:
I'm sorry about the trouble with docker.  Logic was added to skip docker install if `podman` command is found.